fetch.bs

```diff
@@ -2164,6 +2175,15 @@ otherwise, it is unset.
 <a for=request>done flag</a>, and <a for=request>timing allow failed flag</a> are used as
 bookkeeping details by the <a for=/>fetch</a> algorithm.
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-id>request id</dfn> which is a unique
+string automatically set when the <a for=/>request</a> is created.
+
+Note: the [=request/request id=] is used by WebDriver-BiDi. It remains constant
+across all requests resultsing from a redirect of an initial request. When a
+request is [=request/cloned=], the created request gets a unique
+[=request/request id=]. [[!WEBDRIVER-BIDI]]
```

Comment:
These two statements are contradictory. You probably have to manage this a bit more explicitly throughout the specification.